internal/server/data/migrations.go

```diff
@@ -460,3 +463,39 @@ DROP SEQUENCE IF EXISTS password_reset_tokens_id_seq CASCADE;
 		},
 	}
 }
+
+func addDefaultOrganization() *migrator.Migration {
+	return &migrator.Migration{
+		ID: "2022-08-10T13:35",
+		Migrate: func(tx *gorm.DB) error {
+
+			stmt := `
+INSERT INTO organizations(id, name, created_at, updated_at)
+VALUES (? , ?, ?, ?);
+`
```

Comment:
We don't want to use `data.CreateOrganization` here, because that can change over time, which can break this migration.